pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -2114,7 +2112,6 @@ void Object_endTypeContext(JSOBJ Py_UNUSED(obj), JSONTypeContext *tc) {
         NpyArr_freeLabels(GET_TC(tc)->columnLabels,
                           GET_TC(tc)->columnLabelsLen);
         GET_TC(tc)->columnLabels = NULL;
-        if (tc->type == JT_BIGNUM) free(GET_TC(tc)->bigNum_bytes);
         PyObject_Free(GET_TC(tc)->cStr);
```

Comment:
@WillAyd This looks like it frees up `cStr`

Reply:
I don't think that is getting hit here. If it would then you would be double freeing those bytes and get a segfault anyway

Reply:
Sorry I meant 
``` C
PyObject_Free(GET_TC(tc)->cStr);
```
I think `Object_endTypeContext` gets executed in 
https://github.com/pandas-dev/pandas/blob/e6e08890cc8bd1b162b920cf5a526a433bab8b30/pandas/_libs/src/ujson/lib/ultrajsonenc.c#L1112

Reply:
Can you check that the free on cStr gets hit there? If so then good to go

Reply:
Checked just now - it does!